Uplift/Views/Cells/GymWeekCell.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  GymWeekCell.swift
+//  Uplift
+//
+//  Created by Phillip OReggio on 10/6/19.
+//  Copyright © 2019 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+import SnapKit
+
+class GymWeekCell: UICollectionViewCell {
+
+    // MARK: - Views
+    private var dayLabel: UILabel!
+    private var backCircle: UIView!
+
+    // MARK: - Info
+    /// Weekday this cell represents
+    var day: WeekDay?
```

Comment:
When would day ever be evaluated to nil in this file since the only usage of it is through the configure function which takes in a weekday that isn't optional?